vdyp-tools/src/vdyp-browse-usages.bat

```diff
@@ -0,0 +1 @@
+C:\Users\mjunkin\AppData\Local\Programs\Python\Python312\python.exe C:\Users\mjunkin\PycharmProjects\vdyp\browse_usages.py %~1 %~2
```

Comment:
Location of the python executable should also be parameterized if this is going to live in the repo.